itest/lnd_funding_test.go

```diff
@@ -1250,3 +1251,201 @@ func deriveFundingShim(ht *lntest.HarnessTest, carol, dave *node.HarnessNode,
 
 	return fundingShim, chanPoint
 }
+
+// testChannelFundingWithUnstableUtxos tests channel openings with restricted
+// utxo selection. Internal wallet utxos might be restricted due to another
+// subsystems still using it therefore it would be unsecure to use them for
+// channel openings. This test focuses on unconfirmed utxos which are still
+// being used by the sweeper subsystem hence should only be used when confirmed.
+func testChannelFundingWithUnstableUtxos(ht *lntest.HarnessTest) {
+	if ht.IsNeutrinoBackend() {
+		ht.Skipf("skipping test for neutrino backend because it " +
```

Comment:
hmm makes me wonder if we should add a check to disable using unconfirmed utxos for nodes running neutrino.

Reply:
Ohh I think we can remove this check from the test and also run it for neutrino because all transactions originating from ourselves are recorded rightfully as unconfirmed transactions. Only transactions sending to our wallet from outside will not be recorded hence also not used before confirmation.